app/src/main/kotlin/cz/covid19cz/erouska/ui/dashboard/DashboardFragment.kt

```diff
@@ -121,6 +121,10 @@ class DashboardFragment : BaseFragment<FragmentPermissionssDisabledBinding, Dash
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        if (BuildConfig.DEBUG) {
```

Comment:
Když budeme posílat verzi testerům, tak to bude release varianta a ne debug. Jsi si jistý že tohle je jenom pro debug? (pro vývojáře, ne pro testery)

Reply:
Právěže jsme se Štěpánem chtěli aby to bylo právě pro testery, aby alespoň viděli přechody některých stavů ve flow, ale bylo by dost nešťastné, kdybychom omylem zapomněli odebrat tlačítka (proto pro debug, nevěděl jsem, že testeři dostávají release config)

Reply:
Můžeš to udělat pro DEV flavor místo DEBUG configu.